### PR TITLE
new button to close connection

### PR DESCRIPTION
**Pull request by Gyssim**

### CODE REVIEW

File: src/ch/imedias/rsccfx/localization/Strings.java [hunk 1]
```diff
@@ -1,13 +1,12 @@
 package ch.imedias.rsccfx.localization;
 
 /**
- *  Code generated at 11:22:30 25.05.2017
+ *  Code generated at 10:14:06 27.05.2017
```
Comment (l-marchesi) [inline src/ch/imedias/rsccfx/localization/Strings.java:4]: Not quite sure but shouldn't this file be excluded from pushing?
@martinfrancois 
If so, please delete it :)

File: src/ch/imedias/rsccfx/view/RsccRequestPresenter.java [hunk 2]
```diff
@@ -131,8 +139,7 @@ public void initSize(Scene scene) {
     view.supporterDescriptionLbl.prefWidthProperty().bind(scene.widthProperty().divide(3));
     view.supporterInnerPane.prefWidthProperty().bind(scene.widthProperty().divide(3).multiply(2));
     view.reloadKeyBtn.prefHeightProperty().bind(view.generatedKeyFld.heightProperty());
-    view.generatedKeyFld.prefWidthProperty().bind(scene.widthProperty()
```
Comment (l-marchesi) [inline src/ch/imedias/rsccfx/view/RsccRequestPresenter.java:134]: Did this get delete by you or by merging it from develop?

File: src/ch/imedias/rsccfx/view/RsccRequestView.java [hunk 3]
```diff
@@ -64,8 +63,8 @@
   private final KeyUtil keyUtil;
 
   Button reloadKeyBtn = new Button();
+  final Button disconnectBtn = new Button();
 
-  private Pane emptyPane = new Pane();
```
Comment (l-marchesi) [inline src/ch/imedias/rsccfx/view/RsccRequestView.java:68]: It seems a bit weird to me, that the pane is "lost"... Don't we need it to ensure a good looking gui? @JenniferMue & @martinfrancois

**Review by l-marchesi — CHANGES_REQUESTED** [review]
Some smallers things to clarify, will boot into Lernstick and check how it looks

File: src/ch/imedias/rsccfx/view/RsccRequestView.java (hunk 3 above)
Comment (Gyssim) [reply src/ch/imedias/rsccfx/view/RsccRequestView.java:68]: It looked the same without the empty pane. I had an issue that the empty Pane overlapped my button, so I asked Francois and he said I can delete it

Comment (l-marchesi) [reply src/ch/imedias/rsccfx/view/RsccRequestView.java:68]: Okay, great! then it is alright :)

**Comment by l-marchesi** [comment]
Did fix checkstyle errors. Now checking if it looks good

**Comment by l-marchesi** [comment]
I tried to establish a connection between two of my computers, but it does not work. When changing back to branch develop it does. So something seems to be wrong on this branch.

**Comment by l-marchesi** [comment]
Okay, forget about my last comment.
I did redo all the steps I did before and it works now. However, really strange.

**Review by l-marchesi — APPROVED** [review]

**Pull request merged**